package.json

```diff
@@ -31,5 +31,8 @@
             "pre-commit": "npm run test-backend && npm run prettier:check"
         }
     },
-    "dependencies": {}
+    "dependencies": {
+        "@types/react-router-dom": "^5.1.6",
+        "react-router-dom": "^5.2.0"
```

Comment:
I don't think this was intentional - we don't need react-router-dom in the runtime folder. Please revert this change (presumably package-lock.json as well).